### DISCUSSION TITLE
2024-01-25

### DISCUSSION THREAD

**Discussion by yutingzhao1991** [ant-design/ant-design-web3#538] [中文社区双周会]
上次双周会遗留待办：

https://github.com/ant-design/ant-design-web3/discussions/480#discussioncomment-8092020

**Comment by yutingzhao1991** [comment]
提议：
- 交互稿设计 review
- 我们的课程和 https://github.com/WTFAcademy/WTF-Dapp 课程的协作方式

**Comment by yutingzhao1991** [comment]
提议：
- 看看双周会时间要不要调整。 @kiner-tang

**Reply by kiner-tang** [reply]
个人希望调整一下，现在周会的时间刚好跟我们公司每周组会的时间冲突了，看看周三下午或周五下午可以不，看看其他人的意见

**Reply by yutingzhao1991** [reply]
@kiner-tang 双周会改到周三下午四点你这边如何？

**Reply by kiner-tang** [reply]
我这边可以

**Comment by yutingzhao1991** [comment]
提议：
- 讨论简单模式下 footer 和 guide 如何显示 https://github.com/ant-design/ant-design-web3/pull/532

**Comment by yutingzhao1991** [comment]
会议纪要：
- @gin-lsl solidity adapter 基本已经完成，还有一个构建的 ts 问题 @yutingzhao1991 帮忙一起看看
- @Likang0122 继续调研支付组件，补充 dapp 场景，整理需求提给设计师
- @LCJove 写一个脚本来自动同步 https://cryptologos.cc/ 上面的 icon
- 双周会时间暂定改为周三下午四点
- Ant Design Web3 课程明天发布一个版本，和 WTF-dapp 课程各自独立维护

设计相关问题再单独拉会沟通。

**Reply by yutingzhao1991** [reply]
按照 antd 默认风格优化后的设计稿这周会给出
